deploy/manifests/prod/us-east-2/tenant/storetheindex/indexer-1-identity.encrypted

```diff
@@ -0,0 +1,22 @@
+{
+	"data": "ENC[AES256_GCM,data:gMTrwBeT8M1FpliLFp3NBd7paAS+8JoemI/G+TxOMW8x2ERZi8wUALYk6p3Qv4yRRt6iuB93+evRrvuMZhfg+wGNUdw=,iv:49tzBmjrf8Nta6pgdceVRgv07aKIPqhKrFKKiKn7PVM=,tag:uL/CSXdJ0HsWRPbj4mjmFQ==,type:str]",
```

Comment:
a readme / comment somewhere on how to regenerate these secrets would be great

Reply:
A quick guide added [here](https://github.com/filecoin-project/storetheindex/pull/340/commits/639cd3f29eaae20eade4f528b02eb2cbf975ebf8). I will flesh the docs out once the deployment manifests stabilise.